src/environments/environment.ts

```diff
@@ -9,8 +9,8 @@ import env from './.env';
 export const environment = {
   production: false,
   version: env.npm_package_version + '-dev',
-  fineractPlatformTenantId: 'default',  // For connecting to server running elsewhere update the tenant identifier
-  baseApiUrl: 'https://demo.openmf.org',  // For connecting to server running elsewhere update the base API URL
+  fineractPlatformTenantId: 'mobile',  // For connecting to server running elsewhere update the tenant identifier
```

Comment:
Hi, @agamvrinos do not include environment.ts in your PR. use git add src/app/users instead.

Reply:
Ah yes. I missed that while rebasing. Fixed